sickchill/oldbeard/providers/kat.py

```
@@ -115,15 +115,11 @@
 
                             if self.confirmed and not result.find(class_="ka-green"):
                                 if mode != "RSS":
-                                    logger.debug("Found result " + title + " but that doesn't seem like a verified result so I'm ignoring it")
-                                continue
-
-                            torrent_size = result("td")[labels.index("size")].get_text(strip=True)
-                            size = convert_size(torrent_size) or -1
+                                    logger.debug(_("Found result " + title + " but that doesn't seem like a verified result so I'm ignoring it"))
 
                             item = {"title": title, "link": download_url, "size": size, "seeders": seeders, "leechers": leechers, "hash": torrent_hash}
```

Ruff (0.8.2)

Undefined name `size`

(F821)